src/OpenRiaServices.Server.EntityFrameworkCore/Framework/TypeDescriptionContextBase.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace OpenRiaServices.Server
+{
+    /// <summary>
+    /// Base class for LTS and EF type description contexts
+    /// </summary>
+    internal abstract class TypeDescriptionContextBase
```

Comment:
efcore has only whan typedescriptor, move code to where it is used and remove file ?